lwatools/imaging/image_model_fitting.py

```diff
@@ -87,38 +110,10 @@ def main(args):
         # we only want the bin nearest to our frequency
         target_bin = np.argmin([abs(args.tx_freq - f) for f in freqs])
 
-        # Build a VisibilityDataSet with this data (lsl.imaging.data.VisibilityDataSet)
```

Comment:
I might leave some of these comments in place. I suppose so long as it's recorded somewhere that VisibilityDataSet is a required "data shape" that's all that matters. It took me a while to wrap my head around the lsl structures.

Reply:
this comment is still there, just in the grid_visibilities function